Add README for TextAlignmentControl component

im3dabasia opened the pull request:
Part of: https://github.com/WordPress/gutenberg/issues/22891
Follow up: https://github.com/WordPress/gutenberg/pull/67371

## What?
This PR adds a README for the TextAlignmentControl component.

im3dabasia committed:
doc: Add README.md for text-alignment-control

im3dabasia commented:
Hey @t-hamano, could you please review this PR when you have a moment?

Thanks! 🙇

t-hamano commented on packages/block-editor/src/components/text-alignment-control/README.md:
```diff
@@ -0,0 +1,51 @@
+
+### Usage
+
+Renders the Text Alignment Component with `left`, `center`, `right`, and `justify` alignment options.
```
```suggestion
Renders the Text Alignment Component with `left`, `center`, and `right` alignment options.
```

`justify` is an available option, but it is not available [by default](https://github.com/WordPress/gutenberg/blob/27bb038623933c51d75d38da5a9c1a1e4be850f5/packages/block-editor/src/components/text-alignment-control/index.js#L45).

t-hamano reviewed (commented):
Thanks for the PR! 

You probably looked at the README for the `TextDecorationControl` component, but I noticed that the heading levels are incorrect.

The correct format should be as follows.

If there is the developer guidelines:

```
# ComponentName

## Development guidelines

### Usage

### Props

#### prop
```

If there is no developer guidelines:

```
# ComponentName

## Usage

## Props

### prop
```

I think there is no developer guidelines for this component, so could you use the _latter_ format and correct the heading levels?

im3dabasia committed:
doc: Corrected headings and removed justify

im3dabasia commented:
Hi @t-hamano,

I’ve removed the justify option as it wasn’t present by default, and I’ve also corrected the headings to use the appropriate # levels. 

Could you please review the PR again and share your feedback?

t-hamano commented on packages/block-editor/src/components/text-alignment-control/README.md:
```diff
@@ -0,0 +1,49 @@
+### `value`
+
+-   **Type:** `String`
+-   **Default:** `left`
+-   **Options:** `left`, `center`, `right`
```
```suggestion
-   **Default:** `undefined`
-   **Options:** `left`, `center`, `right`, `justify`
```

- The default value is `undefined`, not `left`.
- Adds `justify` because the `Options` field actually needs to show all available options.

t-hamano reviewed (commented):
Thanks for the update! Could you make one last change?

im3dabasia committed:
doc: Fix default and options for value prop

im3dabasia commented:
Hey @t-hamano , Thank you very much for the follow up. I have made the requested changes please check.

t-hamano reviewed (approved):
LGTM 👍
